Python/search/fibonacci_search.py

```diff
@@ -0,0 +1,99 @@
+# Fibonacci search implementation in Python3
+
+"""
+Returns the index of a key in a sorted array or returns -1 if it is not contained
+"""
+
+def FibonacciSearch(arr, x, n):
+    """
+    Inputs-
+    arr: A sorted array in which we're going to search for the key
+    x: The number to be searched for
+    n: Size of the array
+
+    Output-
+    Single integer
+    The index of the key if it exists in the array.
+    else -1
+    """
```

Comment:
Remove the redundant comments

Reply:
I have kept for input one so that one got to know for what the argument is, and removed for output one as I have mentioned 'return type' above. However, changes done!